lib/transports/polling.js

```diff
@@ -237,7 +246,11 @@ Polling.prototype.send = function (packets) {
 
   var self = this;
   parser.encodePayload(packets, this.supportsBinary, function(data) {
-    self.write(data);
+    var compress = packets.some(function(packet) {
+      var options = packet.options || {};
```

Comment:
I think here it would definitely be faster to do:

```js
return options && options.compress;
```

And also maybe ensure we pass `compress: true`? Having defaults in a low-level API like this is kinda weird.